src/App.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+let todoList = [ {
+  id: 1,
+  title: "Complete Assignment",
+}, 
+{
+  id: 2,
+  title: "Clean Room",
+}];
+
+function App() {
+  return (
+    <div style={{ textAlign: 'center' }}>
+      <header>
+        <h1>Todo List</h1>
+        </header>
+        <ul>
+          {todoList.map(function (item) {
+          return <li key = {item.id}>{item.title}</li>;
+        })
```

Comment:
Nice job here!

Next time try an implicit return. It makes the code a bit cleaner.
Like this:
`{todoList.map(todo => (<li key={todoList.id}>
        {todo.title}</li>))}`

You see how I'm not calling `return` here?